src/routes/auth.routes.js

```diff
@@ -1,18 +1,21 @@
 import express from 'express';
 import { signupValidation, loginValidation } from '../utils/authentication';
-
 import {
-  signup,
-  login,
-  changePassword,
-  deleteUser,
-} from '../controllers';
+  validationErrors,
+  requireAllfields,
+  requireCredentials,
+} from '../middleware/authMiddlewares';
```

Comment:
u can just import from '../middleware' now because u are exporting it from index